assets/js/blocks/featured-product/style.scss

```
@@ -1,10 +1,16 @@
+.wp-block-woocommerce-featured-product {
+	border-color: transparent;
+	overflow: hidden;
+	color: #fff;
+	background-color: #000;
+}
+
 .wc-block-featured-product {
 	position: relative;
-	background-color: $gray-900;
 	background-size: cover;
 	background-position: center center;
 	width: 100%;
-	margin: 0 0 1.5em 0;
+	margin: 0 0 0 0;
```

Should we use the shorthand here?
```suggestion
	margin: 0;
```